specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/virtualNetworkGateway.json

```diff
@@ -4447,6 +4729,259 @@
       },
       "description": "Start packet capture parameters on virtual network gateway."
     },
+    "ExpressRouteFailoverAllTestDetails": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/ExpressRouteFailoverTestDetails"
+      },
+      "description": "Gateway Failover All Test Details"
+    },
+    "ExpressRouteFailoverTestDetails": {
+      "type": "object",
+      "properties": {
+        "peeringLocation": {
+          "type": "string",
+          "description": "Peering location of the test"
+        },
+        "circuits": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExpressRouteFailoverCircuitResourceDetails"
+          },
+          "description": "All circuits in the peering location"
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the test",
+          "enum": [
+            "NotStarted",
+            "Starting",
+            "Running",
+            "StartFailed",
+            "Stopping",
+            "Completed",
+            "StopFailed",
+            "Invalid",
+            "Expired"
+          ],
+          "x-ms-enum": {
+            "name": "FailoverTestStatus",
+            "modelAsString": true
+          }
+        },
+        "startTime": {
+          "type": "string",
+          "description": "Time when the test was started"
+        },
+        "endTime": {
+          "type": "string",
+          "description": "Time when the test was completed"
+        },
+        "connections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExpressRouteFailoverConnectionResourceDetails"
+          },
+          "description": "All connections to the circui in the peering location"
+        },
+        "testGuid": {
+          "type": "string",
+          "description": "The unique GUID associated with the test"
+        },
+        "testType": {
+          "type": "string",
+          "description": "The type of failover test",
+          "enum": [
+            "SingleSiteFailover",
+            "MultiSiteFailover",
+            "All"
+          ],
+          "x-ms-enum": {
+            "name": "FailoverTestType",
+            "modelAsString": true
+          }
+        },
+        "issues": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of all issues with the test"
+        }
+      },
+      "description": "ExpressRoute failover test details"
+    },
+    "ExpressRouteFailoverCircuitResourceDetails": {
+      "type": "object",
+      "properties": {
+        "nrpResourceUri": {
+          "type": "string",
+          "description": "NRP Resource URI of the circuit"
+        },
```

Comment:
Is this is a fully qualified resource id it should have "arm-id" format.

Reply:
Added